client/src/NetworkClient.ts

```diff
@@ -1,3 +1,4 @@
+import { TransactionReceipt } from '@nimiq/core-web';
```

Comment:
Have you checked the size of the built script compared to the old size? Does it really _only_ include the TransactionReceipt, or much more? Or not even, because it's just used as a type?

Reply:
No difference!